SPEC.md

```diff
@@ -30,20 +30,20 @@ Type text.
 
 ##	Scope (mandatory)
 
-Specifies procedures for constructing and conveying artifact trees, directed acyclic graphs (DAGs), and other related 
+Specifies procedures for constructing and conveying artifact dependency graphs (ADGs), and other related
 data structures for artifacts.  Including but not limited to:
 
 - formats for artifact identifiers
 - formats for specifying graph relationships between artifacts
-- manner of embedding identifiers for artifact trees, directed acyclic graphs (DAGs), and other related 
+- manner of embedding identifiers for ADGs, and other related
 data structures in artifacts of various types
+- guidance on metadata which references ADGs, and other related data structures
 - guidance for build tools for:
-  - constructing artifact artifact trees, DAGs, and other related data structures
-  - conveying artifact trees, DAGs, and other related data structures
-  - embedding identifiers for artifact trees, DAGs, and other related data structures ids in artifacts
-  - manners of conveyance of artifact trees, DAGs, and other related data structures
-  - descriptions of use cases for which artifact trees, directed acyclic graphs, and other related data structures may be used
-  - guidance on metadata which references artifact trees, DAGs, and other related data structures
+  - constructing ADGss, and other related data structures
+  - conveying ADGs, and other related data structures
+  - embedding identifiers for ADGs, and other related data structures ids in artifacts
+  - manners of conveyance of ADGs, and other related data structures
+  - descriptions of use cases for which directed acyclic graphs, and other related data structures may be used
```

Comment:
Swap "directed acyclic graphs" for ADG as ADGs are defined as DAGs?

Reply:
Fixed in latest commit.